WordPress/src/main/java/org/wordpress/android/ui/mysite/ImprovedMySiteFragment.kt

```diff
@@ -178,6 +178,11 @@ class ImprovedMySiteFragment : Fragment(),
                 }
             }
         })
+        viewModel.onScrollTo.observe(viewLifecycleOwner, {
+            it?.applyIfNotHandled {
+                recycler_view.smoothScrollToPosition(this.second)
```

Comment:
Maybe we should try to set an offset here by using `LinearLayoutManager::scrollToPositionWithOffset`. Again, I haven't tested this myself, so let me know what you think.